cg/apps/demultiplex/sample_sheet/api.py

```diff
@@ -160,6 +160,16 @@ def _use_sample_sheet_from_housekeeper(
                 "would have copied it to sequencing run directory"
             )
             return
+
+        try:
+            if sample_sheet_path.samefile(run_directory_data.sample_sheet_path):
+                LOG.info(
+                    "Sample sheet from Housekeeper already matches the target location. Skipping."
```

Comment:
I would like to specify that the target directory is the sequencing directory for clarity
```suggestion
                    "Sample sheet from Housekeeper is the same as the sequencing directory sample sheet"
```